pyproject.toml

```diff
@@ -1,20 +1,97 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
+# unpin wheel when https://github.com/pypa/wheel/issues/659 is resolved
```

Comment:
This can be resolved now by using higher version of setup tools and removing the wheel dep